DB_testdata.sql

```diff
@@ -0,0 +1,88 @@
+INSERT INTO "Member" ("F_name", "L_name", "Password", "Gender", "Email", "Phone", "Birth", "Address", "Reg_date", "A_flag", "S_flag", "C_flag") 
+VALUES 
+('Harrison', 'Mckinney', 'Password123', 'Male',   'harrison4356@gmail.com',  '0912345678',  '1990-01-15', '123 Main St, Springfield',   '2024-10-13', TRUE,  FALSE, FALSE),
+('Hudson',   'Smith',    'Passw0rd!',   'Male',   'hudson247@gmail.com',     '0923456789',  '1992-05-20', '456 Elm St, Springfield',    '2024-10-14', FALSE, TRUE,  FALSE),
+('Chris',    'Lee',      'Qwerty123',   'Other',  'chris_lee@gmail.com',      NULL,         '1995-03-10', '789 Maple St, Springfield',  '2024-10-28', FALSE, FALSE, TRUE),
+('Cecilia',   NULL,      '1234Pass',    'Female', 'cecilia643@gmail.com',    '0934567890',  '1988-07-08', '321 Oak St, Springfield',    '2024-10-28', FALSE, FALSE, TRUE),
+('Landon',   'Burke',    'Bur2023!',    'Male',   'landon232@gmail.com',     '0945678901',  '1990-10-25', '654 Pine St, Springfield',   '2024-10-28', FALSE, FALSE, TRUE),
+('妍婷',      '許',       'MyP@ssw0rd',  'Female', 'emily331565@gmail.com',    NULL,         '1999-02-14', '987 Birch St, Springfield',  '2024-10-28', FALSE, FALSE, TRUE),
+('家文',      '何',       'T0mP@ss',     'Male',   'tom1121@gmail.com',       '0956789012',   NULL,        '159 Cedar St, Springfield',  '2024-10-28', FALSE, FALSE, TRUE),
+('家綺',      '溫',       'Sophia!@#',   'Female', 'sophiamiller@gmail.com',  '0967890123',  '2000-09-12', '753 Cherry St, Springfield', '2024-10-14', FALSE, TRUE,  FALSE),
+('Asher',    'Clark',    'AsherClark1', 'Other',  'alex_taylor@gmail.com',   '0978901234',  '1994-06-30', '852 Walnut St, Springfield', '2024-10-17', FALSE, TRUE,  FALSE),
+('瑞池',      '宋',       'Liam2024',    'Male',   'liam_anderson@gmail.com', '0989012345',  '1985-11-05', '951 Willow St, Springfield', '2024-10-28', FALSE, FALSE, TRUE);
+
+
+INSERT INTO "Credit_card" ("Number", "CMID", "Expiry_date", "CVV") 
+VALUES 
+('4532147890123456', 3,  '2025-12-31', '123'),
+('5312345678901233', 4,  '2026-06-30', '456'), 
+('3530111333300505', 5,  '2027-09-15', '789'),
+('5412345678901232', 6,  '2026-03-20', '321'),
+('5212345678901234', 10, '2028-11-10', '654');
+
+
+INSERT INTO "Order" ("CMID", "SMID", "Credit_num", "Time", "Ship_address", "Ship_fee", "Status", "Pay_method", "Tot_price") 
+VALUES
+(10, 9, '5212345678901234', '2024-11-24 09:30:00', '987 Elm Dr, Brockway',           35, 'Shipping',   'Credit card', 1300),
```

Comment:
Do you think we should use Taiwan address?
If want, you can change it

Reply:
No, i think this is fine. or maybe you can ask others opinion in meeting tmr
also i add the `DID` in `Order` and fix problems you have mentioned above